### PR TITLE
[onert] Move nnpkg_test.sh into tests/scripts

### PR DESCRIPTION
**Pull request by hseok-oh**
Move nnpkg_test.sh into tests/script for including in test package

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
@llFreetimell Could you confirm correctness ` tf2circle-value-pbtxt-remote-test` ?

**Comment by hseok-oh** [comment]
@glistening @llFreetimell PTAL

File: compiler/tf2circle-value-pbtxt-remote-test/CMakeLists.txt [hunk 1]
```diff
@@ -141,7 +141,7 @@ add_custom_command(
   COMMAND ${CMAKE_COMMAND} -E echo 'HDF5_EXPORT_ACTION_PATH=\"$<TARGET_FILE:nnkit_HDF5_export_action>\"' >> ${TEST_CONFIG}
   COMMAND ${CMAKE_COMMAND} -E echo 'HDF5_IMPORT_ACTION_PATH=\"$<TARGET_FILE:nnkit_HDF5_import_action>\"' >> ${TEST_CONFIG}
   COMMAND ${CMAKE_COMMAND} -E echo 'MODEL2NNPKG_PATH=\"${NNAS_PROJECT_SOURCE_DIR}/tools/nnpackage_tool/model2nnpkg/model2nnpkg.sh\"' >> ${TEST_CONFIG}
-  COMMAND ${CMAKE_COMMAND} -E echo 'NNPKG_TEST_PATH=\"${NNAS_PROJECT_SOURCE_DIR}/tools/nnpackage_tool/nnpkg_test/nnpkg_test.sh\"' >> ${TEST_CONFIG}
+  COMMAND ${CMAKE_COMMAND} -E echo 'NNPKG_TEST_PATH=\"${NNAS_PROJECT_SOURCE_DIR}/tests/scripts/nnpkg_test.sh\"' >> ${TEST_CONFIG}
```
Comment (seanshpark) [inline compiler/tf2circle-value-pbtxt-remote-test/CMakeLists.txt:144]: compiler codes follow the rule to change one module per PR. can you split the changes?
(1) disable this test (2) move others (3) enable this test

**Comment by hseok-oh** [comment]
@glistening PTAL

**Pull request merged**